pipeline/07-export.R

```diff
@@ -174,13 +175,50 @@ assessment_pin10_prepped <- assessment_pin_prepped %>%
 for (town in unique(assessment_pin_prepped$township_code)) {
   message("Now processing: ", town_convert(town))
 
+  workbook_name <- glue(
+    params$assessment$year,
+    str_replace(town_convert(town), " ", "_"),
+    "Initial_Model_Values_Condo.xlsx",
+    .sep = "_"
+  )
 
   ## 4.1. PIN-Level ------------------------------------------------------------
 
+  # Filter building data to specific township
+  assessment_pin10_filtered <- assessment_pin10_prepped %>%
+    filter(township_code == town) %>%
+    select(-township_code)
+
+  building_coords <- assessment_pin10_filtered %>%
+    tibble::rowid_to_column("building_coord") %>%
+    select(meta_pin10, building_coord)
+
+  pin_sheet_name <- "PIN Detail"
+  bldg_sheet_name <- "Buildings"
```

Comment:
This sheet used to be called `Building (PIN10)`, but unfortunately as I discovered when [updating the res model](https://github.com/ccao-data/model-res-avm/pull/199#discussion_r1473590327), sheet names can't have spaces if we want to link to them via between-sheet hyperlink formulas.